### PR TITLE
Elantra 2021 FW versions

### PR DESCRIPTION
**Pull request by sternm33**

### CODE REVIEW

**Comment by sternm33** [comment]
added another Elantra 2021 fingerprint

**Comment by pd0wm** [comment]
What is your dongle id?

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -617,24 +617,27 @@ class Buttons:
     (Ecu.eps, 0x7d4, None): [b'\xf1\x00TM  MDPS C 1.00 1.00 56340-S2000 8409'],
     (Ecu.fwdCamera, 0x7c4, None): [b'\xf1\x00JFA LKAS AT USA LHD 1.00 1.02 95895-D5000 h31'],
     (Ecu.transmission, 0x7e1, None): [b'\xf1\x816U2V8051\x00\x00\xf1\x006U2V0_C2\x00\x006U2V8051\x00\x00DJF0T16NL0\t\xd2GW'],
-  },
+  }
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:620]:
```suggestion
  },
```

**Comment by adeebshihadeh** [comment]
Closing due to inactivity. Feel free to re-open after addressing the comments and questions.

**Pull request closed without merging**